src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -395,6 +453,20 @@ private static JsonNode parseJsonFromBody(Request req) {
         }
     }
 
+    /**
+     * Utility method to parse generic objects from JSON String and return as list.
+     * @param json The string representation of JSON to be parsed.
+     * @param clazz The class used to map each JSON object extracted.
+     */
+    private static <T> List<T> getPOJOFromJSONAsList(String json, Class<T> clazz) {
+        try {
+            JavaType type = mapper.getTypeFactory().constructCollectionType(List.class, clazz);
+            return mapper.readValue(json, type);
+        } catch (JsonProcessingException e) {
+            return null;
```

Comment:
Swallowing exceptions has caused some issues in some of the JSON parsing code in otp-middleware. It might be better just to throw the exception and let the calling method handle (and log) it.

Reply:
Decided to follow the approach of the other JSON method by using the request and `logMessageAndHalt`